docs/photoniq/event-delivery/getting-started-event-delivery.md

```diff
@@ -0,0 +1,196 @@
+---
+sidebar_position: 10
+title: Getting Started with Event Delivery
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+PhotonIQ Event Delivery is a fully managed service, implemented and optimized by Macrometa engineers to ensure proper functionality. For those who prefer a more hands-on approach, this guide provides information to help you actively manage Event Delivery.
+
+In this guide, you'll learn how to:
+- [Set up your first event stream in the PhotonIQ GDN](#setup-event-stream-in-the-photoniq-gdn)
+- [Subscribe to events and recieve real time updates](#subscribe-to-events-and-recieve-real-time-updates)
+- [Monitor the Event Delivery metrics and health stautus](#monitor-the-event-delivery-metrics-and-health-stautus)
+
+
+The Event Delivery service enables you to stream and filter events from Macrometa [streams](../../streams/), as well as publish events to these streams. Before you proceed, here are some key concepts:
+
+- **Publisher:** An application or service that continuously generates data for consumption by a subscriber. In the Event Delivery Service (EDS), collections in the Macrometa Global Data Network (GDN) act as publishers.
```

Comment:
I think this can be confusing. EDS has a publish API that an application can be used to publish events to streams (gdn collection streams and gdn streams) and an application can subscribe and listen to those events. In such use case, the data is not stored in GDN, but GDN acts like pub-sub. So we cannot say collection as publishers.